sessions/43/tasklist.py

```diff
@@ -0,0 +1,18 @@
+tasks = [
+    [[tc.SetBitrate(200, BaudRate.BaudRate1200), 5], SendLoop, WaitMode.NoWait],
+    [tc.SendBeacon(), Send, WaitMode.Wait],
+
+    [tc.ListFiles(201, '/'), Send, WaitMode.Wait],
+
+    # Telemetry download between session 41 and 43
+    [tc.DownloadFile(202, '/telemetry.current', [i for i in range(0, 1000, 100)]), Send, WaitMode.Wait],
+    [tc.DownloadFile(203, '/telemetry.previous', [i for i in range(2100, 2280, 50)]), Send, WaitMode.Wait],
+
+    # First RadFET experiment triggering
+    [tc.PerformRadFETExperiment(13, 150, 110, 'radfet'), Send, WaitMode.Wait],
```

Comment:
czemu nie włączamy eksperymentu po otrzymaniu beconów? jest jakiś konkretny powód, no bo i tak "na żywo" nie dekodujemy poprzedniej telemetrii. W sensie telemetrię można ściągać w trakcie trwania eksperymentu, samo włączenie jego nie jest długie więc nie ma groźby że nas opóźni to i nie pobierzemy telemetrii, a później bez spiny można pobierać (re-tryować) pliki telemetrii i obserwować becona czy się włączył eksperyment.

Reply:
ten tryb się sprawdził w tej task liście: http://mission.pw-sat.pl/master/sessions/15/

Reply:
OK

Reply:
fixed :heavy_check_mark: